### ISSUE TITLE
pylint/2.17.4: doc/data/messages/t/too-many-lines/good/__init__.py has windows line end

### ISSUE THREAD

**Issue by sandrotosi** [pylint-dev/pylint#8883]
### Bug description

It appears as if `doc/data/messages/t/too-many-lines/good/__init__.py` as windows line ends:

```
$ cat -vte pylint-2.17.4/doc/data/messages/t/too-many-lines/good/__init__.py
__all__ = ["is_palindrome", "main"]^M$
^M$
from is_palindrome import is_palindrome^M$
from main import main^M$
```

this is causing some issues in Debian, where importing the 2.17.4 tarball into our git repo will cause the windows line end to be converted to unix, and then our tools complain the code in the packaging repo is different than upstream :shrug: 

would you be able to convert those line ends to unix? `dos2unix` should do the trick

### Command used

```shell
n/a
```

### Pylint output

```shell
n/a
```

### Expected behavior

n/a

### Pylint version

```shell
n/a
```

**Comment by Pierre-Sassoulas** [comment]
Sure, we're trying to have windows line ending in our tests some of the time, but removing from the doc is fine.

**Comment by sandrotosi** [comment]
> Sure, we're trying to have windows line ending in our tests some of the time, but removing from the doc is fine.

oh are there already tests files with windows line-ending? i would have expected to generated the same error i saw for that doc file, but it's not the case

**Comment by Pierre-Sassoulas** [comment]
Probably others too: https://github.com/pylint-dev/pylint/blob/main/.gitattributes. In particular ``doc/data/messages/u/unexpected-line-ending-format/bad.py`` this one being in the doc too (by design this is what we're checking for).

**Comment by sandrotosi** [comment]
oh so maybe being part of .gitattributes make the other files fine for the debian packaging perspective, thanks!
